docs/articles/documentation/guides/advanced-guides/speed-up-test-execution.md

```diff
@@ -0,0 +1,113 @@
+---
+layout: docs
+title: Create Faster Tests
+permalink: /documentation/guides/advanced-guides/speed-up-test-execution.html
+---
+
+# Speed Up Test Execution
+
+This article describes ways to decrease the execution time of TestCafe tests. It includes the following sections:
+
+* [Run Tests Concurrently](#run-tests-concurrently)
+* [Run Tests in Headless Browsers](#run-tests-in-headless-browsers)
+* [Use Roles for Login](#use-roles-for-login)
+* [Set Test Speed](#set-test-speed)
+* [Run Tests in Local Browsers](#run-tests-in-local-browsers)
+* [Mock Requests](#mock-requests)
+* [Optimize Your Page Model](#optimize-your-page-model)
+
+## Run Tests Concurrently
+
+Enable concurrent mode to run multiple browser instances simultaneously. This speeds up the testing process at the expense of higher resource consumption.
```

Comment:
`This speeds up the testing process at the expense of higher resource consumption.`
It's not correct.

What about such variant?
```js
This speeds up the testing process at the expense of test execution parallelization. 
Note, that using the `concurrency` option requires the more resources that the regular test run. 
Choose the `concurrency` option value based on the capabilities of the test environment.
```

Reply:
How about this:

>Enable concurrent mode to run multiple browser instances simultaneously. 

>With `concurrency` enabled, TestCafe runs tests in parallel, which may decrease suite execution time, but requires more resources from your test environment. Choose the option value based on the test environment performance.